### PR TITLE
PAPILIO-113 Closing Activity

### PR DESCRIPTION
**Pull request by mth-1012**
## General

2 new routes for closing and opening an existing Activity

## Usage

It's assumed that the Activity (i.e. activityId) already existed. If not, error will be thrown identical to getActivity route.

Opening: `/api/activity/open/:activityId`

Closing: `/api/activity/close/:activityId`

## Testing

Manual

### CODE REVIEW

File: src/api/repos/activity-repo.ts [hunk 1]
```diff
@@ -69,10 +69,12 @@ const updateActivity = async (id: number, update: any) => {
         console.log(err);
         throw new BaseError('ORM Sequelize Error', 'There has been an error in updating the Activity', 'updateActivity', httpStatusCode.INTERNAL_SERVER, true);
     });
-    return {
-        success: !!result,
-        activity: result[1][0]
-    };
+    if (returning)
```
Comment (p-carrier) [inline src/api/repos/activity-repo.ts:72]: What is returning? is this to keep track if there is multiple updates on the activity? Why do we need it?

**Comment by p-carrier** [comment]
When the activity does exist, it works great. But isn't it supposed to throw an error when to id does not exist? I'm receiving success for a random number that isn't associated with an activity.

Screenshot 2023-04-01 at 2 35 43 PM

File: src/api/repos/activity-repo.ts (hunk 1 above)
Comment (mth-1012) [reply src/api/repos/activity-repo.ts:72]: that's to indicate if we want the whole object `activity` in the response

**Comment by mth-1012** [comment]
> When the activity does exist, it works great. But isn't it supposed to throw an error when to id does not exist? I'm receiving success for a random number that isn't associated with an activity.
> 
> Screenshot 2023-04-01 at 2 35 43 PM

Let me fix that

**Review by p-carrier — APPROVED** [review]
It's good to merge. There is an error message for a wrong id now. 🚀

**Pull request merged**